caniusepython3/overrides.json

```diff
@@ -25,6 +25,7 @@
     "pil": "https://pypi.python.org/pypi/Pillow",
     "py-bcrypt": "https://code.google.com/p/py-bcrypt/",
     "pycountry": "https://pypi.python.org/pypi/pycountry",
+    "pyodbc": "https://code.google.com/p/pyodbc/",
```

Comment:
Can you change this to http://mkleehammer.github.io/pyodbc/releases.html since development of the project no longer occurs on Google Code?